service/history/shard/context_impl.go

```diff
@@ -970,6 +970,7 @@ func (s *ContextImpl) DeleteWorkflowExecution(
 	branchToken []byte,
 	startTime *time.Time,
 	closeTime *time.Time,
+	closeVisibilityTaskId int64,
```

Comment:
nit: closeVisibilityTaskID

Reply:
or maybe `closeExecutionVisibilityTaskID` to be consistent with the task type name defined here: https://github.com/temporalio/temporal/blob/master/service/history/tasks/close_visibility_task.go#L37

Reply:
Fixed, thanks for catching that!